puppet/modules/slave/manifests/init.pp

```diff
@@ -188,10 +188,6 @@
       ensure   => '1.7.9',
       provider => npm,
     }
-    package { 'phantomjs':
-      ensure   => latest,
```

Comment:
Should we ensure absent perhaps to make sure things don't accidentally count on it?